.drone.starlark

```diff
@@ -601,6 +601,418 @@ config = {
 				}
 			],
 		},
+		'core-api-acceptance-encryption-userkeys-nightly': {
+			'suites': {
+				'apiAuth': 'apiAuth-enc-UK',
+				'apiAuthOcs': 'apiAuthOcs-enc-UK',
+				'apiCapabilities': 'apiCapabilities-enc-UK',
+				'apiComments': 'apiComments-enc-UK',
+				'apiFavorites': 'apiFavorites-enc-UK',
+				'apiMain': 'apiMain-enc-UK',
+				'apiSharees': 'apiSharees-enc-UK',
+				'apiShareManagement': 'apiShareManagement-enc-UK',
+				'apiShareManagementBasic': 'apiShareMgmtBasic-enc-UK',
+				'apiShareOperations': 'apiShareOperations-enc-UK',
+				'apiShareReshare': 'apiShareReshare-enc-UK',
+				'apiShareUpdate': 'apiShareUpdate-enc-UK',
+				'apiSharingNotifications': 'apiSharingNot-enc-UK',
+				'apiTags': 'apiTags-enc-UK',
+				'apiTrashbin': 'apiTrashbin-enc-UK',
+				'apiVersions': 'apiVersions-enc-UK',
+				'apiWebdavLocks': 'apiWebdavLocks-enc-UK',
+				'apiWebdavLocks2': 'apiWebdavLocks2-enc-UK',
+				'apiWebdavMove': 'apiWebdavMove-enc-UK',
+				'apiWebdavOperations': 'apiWebdavOperations-enc-UK',
+				'apiWebdavProperties': 'apiWebdavProperties-enc-UK',
+				'apiWebdavUpload': 'apiWebdavUpload-enc-UK',
+			},
+			'databases': [
+				'mysql:5.7',
+			],
+			'servers': [
+				'latest',
+			],
+			'phpVersions': [
+				'7.1',
+			],
+			'runCoreTests': True,
+			'cron': 'nightly',
+			'extraApps': {
+				'encryption': ''
+			},
+			'extraSetup': [
```

Comment:
IMO this will also need the `configure-app` `extraSetup` block like in `core-api-federation-encryption-userkeys-nightly` because it will not get added from the `extraSetup` in `defaults`